ddtrace/tracer.py

```diff
@@ -80,19 +81,131 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None, contex
         if context_provider is not None:
             self._context_provider = context_provider
 
-    def trace(self, name, service=None, resource=None, span_type=None, ctx=None, span_parent=None):
+    def start_span(self, name, service=None, resource=None, span_type=None):
         """
-        Return a span that will trace an operation called `name`. The context that generated
-        the Span may be provided, as well as the current parent Span.
+        Starts and returns a new ``Span`` representing a unit of work.
 
-        :param str name: the name of the operation being traced
+        :param str name: the name of the operation being traced.
+        :param str service: the name of the service being traced.
+        :param str resource: an optional name of the resource being tracked.
+        :param str span_type: an optional operation type.
+
+        To start a new root span::
+
+            >>> span = tracer.start_span("web.request")
+        """
+        span = Span(
+            self,
+            name,
+            service=service,
+            resource=resource,
+            span_type=span_type,
+        )
+        self.sampler.sample(span)
+
+        # add common tags
+        if self.tags:
+            span.set_tags(self.tags)
+
+        # create a new context for the root span
+        context = Context()
+        context.add_span(span)
+        return span
+
+    def start_child_span(self, name, parent_span, service=None, resource=None, span_type=None):
```

Comment:
What about merging it to `start_span` (`parent_span` being an optional argument), and call `parent_span` `child_of`? (because it can be both a span and a context ; plus it is the same as OT).